docs/anchoring-assets/anchor-platform/sep31/configuration.mdx

```diff
@@ -0,0 +1,368 @@
+assets:
+  - schema: stellar
+    code: USDC
+    issuer: GBBD47IF6LWK7P7MDEVSCWR7DPUWV3NY3DTQEVFL4NAT4AQH3ZLLFLA5
+    distribution_account: GBLSAHONJRODSFTLOV225NZR4LHICH63RIFQTQN37L5CRTR2IMQ5UEK7
+    significant_decimals: 2
+    sep31_enabled: true
+    sep31: {}
+    send: {}
+```
+
+</CodeExample>
+
+The information provided in the `sep31` and `send` objects closely map to the information that will be exposed to the wallet application using the [`GET /info`][sep31-get-info] SEP-31 endpoint. The Anchor Platform also uses this information to validate requests made to your service. We'll populate this information as we go.
+
+Add the following variable to your environment file.
+
+<CodeExample>
+
+```bash
+# dev.env
+SEP31_ENABLED=true
+```
+
+</CodeExample>
+
+Senders should now be able to discover, authenticate, and initiate transactions with your service! Run the following command to start the Anchor Platform.
+
+<CodeExample>
+
+```bash
+docker compose up
+```
+
+</CodeExample>
+
+Check that your API is live.
+
+<CodeExample>
+
+```bash
+curl http://localhost:8080/sep31/info | jq
+```
+
+</CodeExample>
+
+You should get the following.
+
+<CodeExample>
+
+```json
+{
+  "receive": {
+    "USDC": {
+      "enabled": true,
+      "quotes_supported": false,
+      "quotes_required": false
+    }
+  }
+}
+```
+
+</CodeExample>
+
+## Enable the Customer KYC API
+
+Businesses need to collect and validate KYC information on the customers they're facilitating transactions for. Clients determine what KYC information needs to be collected and send that information via a SEP-12 KYC API hosted by the Anchor Platform, but the Anchor Platform never stores personally-identifiable information (PII). Instead, it forwards requests from clients to the business server, and returns the business' responses back to the client, acting as a proxy server.
+
+See the _Anchor Platform KYC API specification_ for details on the endpoints that must be implemented on your business' server.
+
+To make this API available to clients, lets add the service URL to our Stellar Info File.
+
+<CodeExample>
+
+```toml
+# dev.stellar.toml
+DIRECT_PAYMENT_SERVER = "http://localhost:8080/sep31"
+WEB_AUTH_ENDPOINT = "http://localhost:8080/auth"
+KYC_SERVER = "http://localhost:8080/kyc"
+```
+
+</CodeExample>
+
+Lets enable it in our environment too.
+
+<CodeExample>
+
+```bash
+# dev.enva
+SEP12_ENABLED=true
+```
+
+</CodeExample>
+
+Finally, we have to define your business' customer types. Each type of customer requires different a set of KYC information. For example, you can offer your cross-border payments service in two distinct regulatory jurisdictions, so customers in different jurisdictions have different KYC requirements and would be represented using different types.
+
+:::info
+
+Currently, customer types must be mutually exclusive, meaning a customer cannot be more than one type. For example, business' cannot model a tiered KYC model with "basic" and "pro" customer types.
```

Comment:
They technically can, if this types are mutually exclusive. I think it's good idea to elaborate:
"Meaning, user can not be assigned to both 'basic' and 'pro' types"

Reply:
Good point, a user could be both basic and pro as long as pro users can do everything basic can do, so its not a good example. I removed it.